src/simplicial-hott/05-segal-types.rzk.md

```diff
@@ -994,8 +1010,8 @@ arrow.
     ( ( homotopy-to-hom2 A x y f h) ,
       ( total-equiv-family-of-equiv
         ( hom A x y)
-        ( \k -> (f = k))
-        ( \k -> (hom2 A x x y (id-arr A x) f k))
+        ( ̨ -> (f = k))
+        ( ̨ -> (hom2 A x x y (id-arr A x) f k))
```

Comment:
This looks like a mistake

Reply:
Good catch! Thanks.

This is an annoyance involving my method of typesetting LaTeX characters in vsCode. I type \to and then run `Unicode: replace latex` which turns all \k's into whatever that symbol is (though \ k is fine).